package.json

```diff
@@ -19,7 +19,10 @@
       "./test/jestsetup.js"
     ],
     "collectCoverageFrom": [
-      "src/**/*.{js,jsx}"
+      "src/**/*.{js,jsx}",
+      "!src/**/index.jsx",
```

Comment:
Keep in mind this will exclude every `index.jsx` from the `src` tree. So maybe it's better to point the file directly.